src/api/routes/business-routes.ts

```diff
@@ -34,6 +34,8 @@ router.delete('/business/:businessId/removeEmployee/:employeeId', validate(busin
 
 router.delete('/business/:businessId/removeActivity/:activityId', validate(businessSchema.removeActivity), businessController.removeActivity);
 
+router.delete('/business/:businessId', validate(businessSchema.removeBusiness), businessController.removeBusiness);
```

Comment:
I know the HTTP request is DELETE but is it better to make a more descriptive URL? Looks kinda risky to me.

Reply:
with the HTTP method, it makes it secure. At least that's what I can see on other publicly available APIs. This is an example: [strapi](https://github.com/strapi/strapi/blob/98632e10278ad8257bfe1251a01f93dc29986cd2/packages/core/content-manager/server/routes/admin.js#L135-L148) 

There are 3 methods for the same route (GET, PUT, DELETE). 

I always thought that adding more information was redundant, but if you want to have more I can add more.